worlds/tloz/Locations.py

```diff
@@ -0,0 +1,350 @@
+    "Level 1 Compass": 0x54,
+    "Level 1 Boss": 0x35,
+    "Level 1 Triforce": 0x36,
+    "Level 1 Key Drop (Keese Entrance)": 0x72,
+    "Level 1 Key Drop (Moblins)": 0x23,
+    "Level 1 Key Drop (Stalfos Water)": 0x33,
+    "Level 1 Key Drop (Stalfos Entrance)": 0x74,
+    "Level 1 Key Drop (Stalfos Middle)": 0x53,
+    "Level 1 Key Drop (Wallmasters)": 0x45,
+    "Level 2 Item (Magical Boomerang)": 0x4F,
+    "Level 2 Map": 0x5F,
+    "Level 2 Compass": 0x6F,
+    "Level 2 Boss": 0x0E,
+    "Level 2 Triforce": 0x0D,
+    "Level 2 Key Drop (Ropes West)": 0x6C,
+    "Level 2 Key Drop (Moldorms)": 0x3E,
+    "Level 2 Key Drop (Ropes Middle)": 0x4E,
+    "Level 2 Key Drop (Ropes Entrance)": 0x7E,
+    "Level 2 Bomb Drop (Keese)": 0x3F,
+    "Level 2 Bomb Drop (Moblins)": 0x1E,
+    "Level 2 Rupee Drop (Gels)": 0x2F,
+    "Level 3 Item (Raft)": 0x0F,
+    "Level 3 Map": 0x4C,
+    "Level 3 Compass": 0x5A,
+    "Level 3 Boss": 0x4D,
+    "Level 3 Triforce": 0x3D,
+    "Level 3 Key Drop (Zols and Keese West)": 0x49,
+    "Level 3 Key Drop (Keese North)": 0x2A,
+    "Level 3 Key Drop (Zols Central)": 0x4B,
+    "Level 3 Key Drop (Zols South)": 0x6B,
+    "Level 3 Key Drop (Zols Entrance)": 0x7B,
+    "Level 3 Bomb Drop (Darknuts West)": 0x69,
+    "Level 3 Bomb Drop (Keese Corridor)": 0x4A,
+    "Level 3 Bomb Drop (Darknuts Central)": 0x5B,
+    "Level 3 Rupee Drop (Zols and Keese East)": 0x5D,
+    "Level 4 Item (Stepladder)": 0x60,
+    "Level 4 Map": 0x21,
+    "Level 4 Compass": 0x62,
+    "Level 4 Boss": 0x13,
+    "Level 4 Triforce": 0x03,
+    "Level 4 Key Drop (Keese Entrance)": 0x70,
+    "Level 4 Key Drop (Keese Central)": 0x51,
+    "Level 4 Key Drop (Zols)": 0x40,
+    "Level 4 Key Drop (Keese North)": 0x01,
+    "Level 5 Item (Recorder)": 0x04,
+    "Level 5 Map": 0x46,
+    "Level 5 Compass": 0x37,
+    "Level 5 Boss": 0x24,
+    "Level 5 Triforce": 0x14,
+    "Level 5 Key Drop (Keese North)": 0x16,
+    "Level 5 Key Drop (Gibdos North)": 0x26,
+    "Level 5 Key Drop (Gibdos Central)": 0x47,
+    "Level 5 Key Drop (Pols Voice Entrance)": 0x77,
+    "Level 5 Key Drop (Gibdos Entrance)": 0x66,
+    "Level 5 Key Drop (Gibdos, Keese, and Pols Voice)": 0x27,
+    "Level 5 Key Drop (Zols)": 0x27,
+    "Level 5 Bomb Drop (Gibdos)": 0x65,
+    "Level 5 Bomb Drop (Dodongos)": 0x56,
+    "Level 5 Rupee Drop (Zols)": 0x57,
+    "Level 6 Item (Magical Rod)": 0x75,
+    "Level 6 Map": 0x19,
+    "Level 6 Compass": 0x68,
+    "Level 6 Boss": 0x1C,
+    "Level 6 Triforce": 0x0C,
+    "Level 6 Key Drop (Wizzrobes Entrance)": 0x7A,
+    "Level 6 Key Drop (Keese)": 0x58,
+    "Level 6 Key Drop (Wizzrobes North Island)": 0x29,
+    "Level 6 Key Drop (Wizzrobes North Stream)": 0x1A,
+    "Level 6 Key Drop (Vires)": 0x2D,
+    "Level 6 Bomb Drop (Wizzrobes)": 0x3C,
+    "Level 6 Rupee Drop (Wizzrobes)": 0x28
+}
+
+floor_location_game_ids_early = {}
+floor_location_game_ids_late = {}
+for key, value in floor_location_game_offsets_early.items():
```

Comment:
`floor_location_game_ids_early = {key, value + Rom.first_quest_dungeon_items_early for key, value in floor_location_game_offsets_early.items()}